src/components/VideoDetails/VideoDetails.component.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { useParams } from 'react-router';
+import GeneralContext from '../../state/GeneralContext';
+import { StyledVideoDetails } from '../../styled/VideoDetails.styled';
+import RecommnendedBar from './RecommendedBar.component';
+import { getVideoDetails, getVideos } from '../../utils/services';
+
+function VideoDetails() {
+  const { keyword } = useContext(GeneralContext);
+  const { videoId } = useParams();
+  const [recommended, setRecommended] = useState([]);
+  const [videoTitle, setVideoTitle] = useState('');
+  const [videoDescription, setVideoDescription] = useState('');
+
+  useEffect(() => {
+    async function loadVideos() {
+      const response = await getVideos(keyword);
```

Comment:
I would rename this variable to `videosResponse` or something similar like you did the details response

```suggestion
      const videosResponse = await getVideos(keyword);
```